components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_lts.F

```diff
@@ -505,18 +496,20 @@ subroutine ocn_time_integrator_lts(domain,dt)!{{{
     ! --- coarse
     do ie = 1, nEdgesInLTSRegion(2,1)
        iEdge = edgesInLTSRegion(2,1,ie)
-       normalVelocitySecondStage(:,iEdge) = weightOld * normalVelocityCur(:,iEdge) &
-                                            + weightNew * normalVelocityFirstStage(:,iEdge) &
-                                            + weightTend * dt * normalVelocityTend(:,iEdge)
+       normalVelocitySecondStage(:,iEdge) = ( weightOld * normalVelocityCur(:,iEdge) &
+                                             + weightNew * normalVelocityFirstStage(:,iEdge) &
+                                             + weightTend * dt * normalVelocityTend(:,iEdge) ) &
+                                            * (1.0_RKIND - wettingVelocityFactor(:, iEdge))
```

Comment:
I'm confused about why you need this factor here. Have you scaled each of the terms here already? It seems that you are double-scaling at least a few of them.

Reply:
Thanks for your feedback @cbegeman . There is no double scaling, except the one coming from previous applications of the limiter at earlier stages if that was needed. `wettingVelocityFactor` is computed in `ocn_lts_wd` and then used once in the `flux` for the `h` tendency (in the call to `ocn_tend_lts`) and then subsequently where we advance the normal velocity solution. In LTS we have ad-hoc routines to compute the tendencies (`ocn_tend_lts`) due to the splitting, so `ocn_tend_vel` is not used. I think that a double scaling could be happening though for RK4, because [here](https://github.com/E3SM-Project/E3SM/blob/839d1cf38fe80aa59a934f3ec5a2c260f3b8aa72/components/mpas-ocean/src/shared/mpas_ocn_wetting_drying.F#L281) the old velocity is limited and then again multiplied by the factor [here](https://github.com/E3SM-Project/E3SM/blob/839d1cf38fe80aa59a934f3ec5a2c260f3b8aa72/components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_rk4.F#L1249). The tendency is also limited twice, where the provis solution is advanced and in the [call](https://github.com/E3SM-Project/E3SM/blob/839d1cf38fe80aa59a934f3ec5a2c260f3b8aa72/components/mpas-ocean/src/shared/mpas_ocn_tendency.F#L469) to `ocn_tend_vel`, as you mentioned.

Reply:
Thank you for this detailed explanation! That helped me understand what was going on.

Reply:
Of course, thanks for reviewing!